components/lib/calendar/Calendar.js

```diff
@@ -1852,7 +1854,7 @@ export const Calendar = React.memo(
                 setOverlayVisibleState(true);
 
                 overlayEventListener.current = (e) => {
-                    if (!isOutsideClicked(e)) {
+                    if (!isOutsideClicked(e.target)) {
```

Comment:
This line looks incorrect as `isOutsideClicked` takes an `event` and gets its `event.target`?

Reply:
When target is not included, the panel of the first calendar input stays open after making changes in the first input and shifting focus to the second calendar input.

![Screen Shot 2024-09-22 at 15 01 01 PM](https://github.com/user-attachments/assets/e29ac413-6f1e-4914-869e-ccca257a4b7f)

Reply:
OK but if you look at the code 

```js
const isOutsideClicked = (event) => {
            return elementRef.current && !(elementRef.current.isSameNode(event.target) || isNavIconClicked(event.target) || elementRef.current.contains(event.target) || (overlayRef.current && overlayRef.current.contains(event.target)));
        };
```

That means if you pass `e.target` then inside isOutSideClicked they will be calling `target.target` does that make sense?  the Target is a DOM element and it won't have a target property?

Reply:
Thanks for the clarification, I missed that detail. I'll remove the target and push the update. Can I address the other issue in a different PR?

Reply:
noticed missing condition, added a fix to resolve the issue.